core/renderers/zelos/path_object.js

```diff
@@ -87,3 +94,47 @@ Blockly.zelos.PathObject.prototype.updateSelected = function(enable) {
     }
   }
 };
+
+/**
+ * Create's an outline path for the specified input.
+ * @param {string} name The input name.
+ * @return {!SVGElement} The SVG outline path.
+ * @package
+ */
+Blockly.zelos.PathObject.prototype.createOutlinePath = function(name) {
+  if (!this.outlines_[name]) {
+    this.outlines_[name] = Blockly.utils.dom.createSvgElement('path', {
+      'class': 'blocklyOutlinePath',
+      // IE doesn't like paths without the data definition, set empty default
```

Comment:
Do I also need to do this for the calls to create the main, light, and dark paths?

Reply:
Possibly, will dig up the bug that introduced this and try to repro.